src/main/java/com/stripe/model/SourceTransaction.java

```diff
@@ -140,6 +141,43 @@
     String sortCode;
   }
 
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class JpCreditTransferData extends StripeObject {
```

Comment:
What is this diff?

Reply:
A new source type that I'm not sure is ready for release. You should use the same version of the spec that we used to generate the current version of stripe-java to ensure we're not including any new stuff.